CMakeLists.txt

```diff
@@ -136,6 +138,8 @@ else()
 	endif()
 endif()
 
+target_compile_features(libninja PUBLIC cxx_std_11)
```

Comment:
Imho, the better way to handle this is:

```
# Set the explicit standard revision for C and C++ code.
SET(CMAKE_C_STANDARD 11)
SET(CMAKE_C_EXTENSIONS OFF)
SET(CMAKE_C_STANDARD_REQUIRED ON)

SET(CMAKE_CXX_STANDARD 11)
SET(CMAKE_CXX_EXTENSIONS OFF)
SET(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Reply:
I had something very similar to that in an earlier pull request but in a now-resolved comment (Sarcasm reviewed on Sep 19, 2021) on https://github.com/ninja-build/ninja/pull/2020 it was suggested that this was wrong. I have zero expertise with CMake so I don't know what the best practices are.

Reply:
Ah. Alright then, nevermind my suggestion. May well be I've been doing it wrong.